.travis.yml

```diff
@@ -67,5 +67,11 @@ script:
   # this requires the mantid-workbench package to be fixed
   # - xvfb-run --server-args="-screen 0 640x480x24" --auto-servernum mantidworkbench -q -x workbenchshutdown.py
 
+  # Install pytest-cov from pip
```

Comment:
Please move this closer to where all the other installs happen. L41 is right after the environment is activated.